src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example1.txt

```diff
@@ -0,0 +1,29 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="NeedBraces"/>
```

Comment:
Please remove such txt files.
They are not used in template file.

We already migrated this Check to java examples

Reply:
We need update in exampleX.java to show such attention point.